src/app/_schemas/ecosystemProject/FrontMatterSchema.ts

```diff
@@ -1,22 +1,25 @@
 import { z } from 'zod'
 
-import { DynamicBaseDataSchema } from '@/schemas/dynamicDataBaseSchema'
+import { BaseDataSchemaKebabCase } from '@/schemas/dynamicDataBaseSchema'
 
 import { CategorySchema, SubcategorySchema } from './CategorySchemas'
 
-export const EcosystemProjectFrontMatter = DynamicBaseDataSchema.extend({
+// email and full-name are required in the CMS but optional here because some entries still don't have them
```

Comment:
Let's change the data - If I remember correctly, Matt was putting in his own email address for the ones that don't have it.

Reply:
Will do!